neo/bin/prompt.py

```diff
@@ -55,6 +55,47 @@
 FILENAME_PROMPT_HISTORY = os.path.join(PATH_USER_DATA, '.prompt.py.history')
 
 
+class NeoHistory(FileHistory):
+    contract_meta_count = 0
+
+    def append(self, string):
```

Comment:
this is good! (see also the original FileHistory here https://github.com/jonathanslenders/python-prompt-toolkit/blob/313dd70224f508c43b0165fc0bfa0a80c54e6294/prompt_toolkit/history.py)

Reply:
Thank! that's where I got most of the `append` function from.